src/handleTask.js

```diff
@@ -0,0 +1,59 @@
+export const tasks = localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
+
+export default class Task {
+  constructor(description, index) {
+    this.description = description;
+    this.completed = false;
+    this.index = index;
+  }
+}
+
+export const addTask = (description) => {
+  const task = new Task(description, tasks.length + 1);
+  const newTasks = [...tasks, task];
+  localStorage.setItem('tasks', JSON.stringify(newTasks));
```

Comment:
- [ ] Kindly alter your localStorage implementation as the checked status is not being preserved. Whenever I reload the page, it looses track of whether the task was checked or not. Please make sure the checked status of the task is preserved.

Reply:
Okay Good dear, for this feature, kindly it's mango features i have to implement on folowing work of the same project and if you go on the branch "**state**"  you'il get everything now, i alredy work on it

Reply:
Please note that the student has implemented this change in another branch, so you don't need to check this feature anymore. I've resolved this issue with him over a meeting and this logic is working perfectly.